gravitee-apim-console-webui/src/management/api/history-v4/api-history-v4.component.spec.ts

```diff
@@ -0,0 +1,141 @@
+/*
+ * Copyright (C) 2015 The Gravitee team (http://gravitee.io)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import { ComponentFixture, TestBed } from '@angular/core/testing';
+import { HttpTestingController } from '@angular/common/http/testing';
+import { NoopAnimationsModule } from '@angular/platform-browser/animations';
+import { MatIconTestingModule } from '@angular/material/icon/testing';
+import { HarnessLoader } from '@angular/cdk/testing';
+import { TestbedHarnessEnvironment } from '@angular/cdk/testing/testbed';
+import { ActivatedRoute } from '@angular/router';
+import { MatTableModule } from '@angular/material/table';
+import { MatCardModule } from '@angular/material/card';
+import { MatTableHarness } from '@angular/material/table/testing';
+
+import { ApiHistoryV4Component } from './api-history-v4.component';
+import { DeploymentsTableComponent } from './deployments-table/deployments-table.component';
+
+import { fakeEvent, fakeEventsResponse, Pagination, SearchApiEventParam } from '../../../entities/management-api-v2';
+import { CONSTANTS_TESTING, GioTestingModule } from '../../../shared/testing';
+import { GioTableWrapperModule } from '../../../shared/components/gio-table-wrapper/gio-table-wrapper.module';
+import { GioTableWrapperHarness } from '../../../shared/components/gio-table-wrapper/gio-table-wrapper.harness';
+
+describe('ApiHistoryV4Component', () => {
+  const API_ID = 'an-api-id';
+
+  let fixture: ComponentFixture<ApiHistoryV4Component>;
+  let loader: HarnessLoader;
+  let httpTestingController: HttpTestingController;
+
+  beforeEach(async () => {
+    TestBed.configureTestingModule({
+      imports: [NoopAnimationsModule, MatIconTestingModule, GioTestingModule, MatTableModule, MatCardModule, GioTableWrapperModule],
```

Comment:
to avoid redefining all the component imports, we generally try to use the module (`ApiHistoryV4Module`) here and only have the imports linked to the test.

Reply:
Good point :) I left only NoopAnimationsModule and two test modules